server/views/pages/bookedtoday/arrivals/confirmArrival.njk

```diff
@@ -1,62 +1,48 @@
-                                <dt class="govuk-summary-list__key">
-                                    Prison number
-                                </dt>
-                                <dd class="govuk-summary-list__value" data-qa="prison-number">
-                                    {{ data.prisonNumber }}
-                                </dd>
-                            </div>
-                        {% endif %}
-                        {%  if data.pncNumber %}
-                        <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">
-                                PNC number
-                            </dt>
-                            <dd class="govuk-summary-list__value" data-qa="pnc-number">
-                                {{ data.pncNumber }}
-                            </dd>
-                        </div>
-                        {%  endif  %}
-                    </dl>
+                    <span class="govuk-caption-xl"> Confirm a prisoner's arrival </span>
+                    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{ pageHeading }}</h1>
+                    <p data-qa="message-text"> {{ data.firstName }} {{ data.lastName }} {{ information }} </p>
                 </div>
             </div>
-            <div>
+              <div class="govuk-grid-row govuk-!-padding-left-3 govuk-!-margin-top-4 govuk-!-margin-bottom-8 flex" >
+                <div class="column-width-40 govuk-!-padding-right-9">
+                    <h2 class="govuk-heading-m govuk-!-padding-right-9">Details from Person Escort Record</h2>
+                    <div class = "prisoner-record-detail prisoner-head">
+                        {{ prisonerSummaryDetail(data, "per-record" ) }}
+                    </div>
+                </div>
+                {% if existingRecord %}
+                    <div class=" border-left-grey govuk-!-padding-left-9 column-width-60">
+                        <h2 class="govuk-heading-m">Details from existing prisoner record</h2> 
+                        <div class="prisoner-record-detail flex">
+                            {{ prisonerSummaryDetail(data.potentialMatches[0], "existing-record") }}
+                            <img src="{{ '/prisoner/' + data.potentialMatches[0].prisonNumber + '/image' if data.potentialMatches[0].prisonNumber else '/assets/images/placeholder-image.png' }}" 
+                                class="prisoner-image-transfer govuk-!-margin-top-2" 
+                                alt="{{ data.lastName }}, {{ data.firstName }}" 
+                                data-qa="prisoner-image" 
+                            />
+                        </div>
+                    </div>
+                {% endif %}
+            </div> 
+            <div>   
                 {{ govukButton({
                     classes: "govuk-button govuk-!-margin-bottom-6",
                     href: "/prisoners/" + data.id + "/sex",
```

Comment:
no need to integration test button or link below as the hrefs are non-changing, apart from the data.id